web/src/layouts/BlogLayout/BlogLayout.tsx

```diff
@@ -43,13 +41,13 @@ const BlogLayout = ({ children }: BlogLayoutProps) => {
             </li>
             <li>
               <Link className="rw-button" to={routes.about()}>
-                About
+                Reisgezelschap
               </Link>
             </li>
           </ul>
         </nav>
       </header>
-      <main className="mx-auto max-w-xl">{children}</main>
+      <main className="max-w-screen mx-auto p-4">{children}</main>
```

Comment:
Was it a conscious decision to use the whole width of the screen for listing the posts? It will look like this on my external monitor: 😅 
![afbeelding](https://github.com/drikusroor/ohayo-goededagu/assets/8208970/cf65b760-55b9-401d-bcb6-6274a9e3b33f)